packages/bundlers-tests/templates/configs/nginx.conf

```diff
@@ -0,0 +1,17 @@
+server {
```

Comment:
Could we use node [serve](https://www.npmjs.com/package/serve) instead of nginx?